### DISCUSSION TITLE
Compiling Error

### DISCUSSION THREAD

**Discussion by Soheilprs** [Cyfrin/foundry-full-course-cu#441] [Q&A]
Hi, i copy a project from another source to my computer, now when i run the commands for build or test, it says compilation skipped
![Screenshot from 2023-07-23 11-37-10](https://github.com/Cyfrin/foundry-full-course-f23/assets/106927616/c6ed2b41-08f0-44c2-b911-608a1eaa651b)
![Screenshot from 2023-07-23 11-36-57](https://github.com/Cyfrin/foundry-full-course-f23/assets/106927616/ac6a0abe-b99d-494e-819d-d112bfec0849)

**Comment by Niferu** [comment]
Hello @Soheilprs 

It says skipped as there was no recent changes in your directory contract file. Just try to change anything in your contract and run it then it should work or just force it by using following command `forge compile --force`.
